spec/features/admin/dashboard_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe 'Admin Dashboard' do
+  # include ApplicationHelper
+  before :each do
+    mock_admin_login
+  end
+
+  describe 'Page Display' do
+    it 'diplays "Admin Dashboard"' do
+      visit admin_path
+      
+      expect(page).to have_content("Admin Dashboard")
+      expect(page).to_not have_button("Admin")
```

Comment:
We don't need this expectation per my comment above. I'd prefer to still display the link to the admin dashboard even if they're already there.